restifizer/RestifizerResponse.h

```diff
@@ -0,0 +1,22 @@
+//
+//  RestifizerResponce.h
+//  Restifizer
+//
+//  Created by Nikita Pankiv on 4/29/16.
+//  Copyright © 2016 VedideV. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "RestifizerError.h"
+
+@class RestifizerRequest;
+
+@interface RestifizerResponse : NSObject
+
+@property (nonatomic, strong) NSData *contentData;
```

Comment:
@qaz12y, might be better to use NSObject (processed by NSJSONSerialization, because all responses will be jsoned objects) instead of NSData.